documentation/plugins.md

```diff
@@ -823,7 +823,7 @@ The `copy_resources` plugin will do this for you at build-time.
 ### Filtering files
 
 With the `filter_resources` plugin, it is possible to replace placeholder values of type `${version}` with actual values at build-time.
-The actual values are attributes of the `project` object, so `${version}` will be replaced with the value of `project.version`.
+The actual values are attributes and properties dict of the `project` object, so `${version}` will be replaced with the value of `project.version` and `${some_property}` by `project.get_property('some_property')`.
```

Comment:
Change to: `The actual values are attributes and the properties dict` without an article, the wording is rough.

Reply:
`The actual values are attributes `project` object and project properties` ?

Reply:
More like:

```
The actual values are the attributes and the properties dict of the `project` object. So `${version}` will be replaced with the value of `project.version` and `${some_property}` by `project.get_property('some_property')`.
```

Mainly, the article `the` makes the paragraph read more fluently, makes sense?

Reply:
@esc thanks. Fixed